### ISSUE TITLE
iPhone 12 Pro Max not being recognized

### ISSUE THREAD

**Issue by castrolem** [ptelad/react-native-iphone-x-helper#32]
## Issue
We have an app that has support for all iPhone sizes, and we use the plug in to set a consistent `marginTop` layout from the notch on one of our UI components.

## How to reproduce
* Install npm package on a `create-react-native-app` project.
* `console.log(isIphoneX())` on `App.js`.
* Run a new iPhone 12 Pro Max simulator.
* console output is always `false`.

**Comment by osikes** [comment]
Seeing the same thing + 1

**Comment by ptelad** [comment]
Are you using the latest 1.3.0?

**Comment by castrolem** [comment]
> Are you using the latest 1.3.0?

Yup, using `^1.3.1`

**Comment by ptelad** [comment]
What value are you getting from Dimensions.get('window').width and height?

**Comment by castrolem** [comment]
@ptelad just tested, `{height: 926, width: 428 }` and asked around about it when we upgraded to 1.3.1 that fixed the issue for us. (sorry, another team in my company ended up taking this one)

**Comment by osikes** [comment]
1.3.1 resolved for me as well.

**Comment by daaashleywch** [comment]
@castrolem The original comment points this out as a problem but semantically, based on the library name, this would be correct behavior. The iPhone 12 is not an iPhone X. This gave me a lot of confusion just now while updating my orgs legacy app. Maybe you could consider adding a custom isIphone12() function?

**Comment by castrolem** [comment]
@daaashleywch I agree, the name of the library hasn't aged well, but the description of the library is accurate
>  A library to help you design your react-native app for notched iPhones

And the 12 Pro Max is a notched iPhone, previous versions (before 1.3.1) worked with other 12 versions, just the Pro Max had the issue, no longer an issue after 1.3.1. 👍
